test/e2e_node/device_plugin_test.go

```diff
@@ -103,6 +105,18 @@ func testDevicePlugin(f *framework.Framework, pluginSockDir string) {
 				return nodes == 1
 			}, time.Minute, time.Second).Should(gomega.BeTrue())
 
+			v1alphaPodResources, err = getV1alpha1NodeDevices()
+			framework.ExpectNoError(err)
+
+			v1PodResources, err = getV1NodeDevices()
+			framework.ExpectNoError(err)
```

Comment:
Please add a message explaining which operation failed here.

Reply:
Would also be a good enhancement for other such calls.

Reply:
Updated, Thanks.

Reply:
I would be inclined to split other such fixes to a separate PR to avoid expanding the scope of this PR.